src/assignment/views/AssignmentOverview.tsx

```diff
@@ -358,12 +358,18 @@ const AssignmentOverview: FunctionComponent<AssignmentOverviewProps> = ({
 				break;
 			case 'duplicate':
 				try {
+					if (!user?.profile?.id) {
+						ToastService.danger(
+							'Je moet ingelogd zijn om een opdracht te kunnen dupliceren'
+						);
```

Comment:
Extract translations

Reply:
fixed in https://github.com/viaacode/avo2-client/pull/1782